README.md

```diff
@@ -9,17 +9,30 @@ This is the Glitch equivalent of running `create-probot-app` to generate a new p
 
 To get your own Glitch-hosted Probot up-and-running:
 
-1. Head on back to our [Devlopment Docs](https://probot.github.io/docs/development/#configuring-a-github-app) to configure your GitHub App.
```

Comment:
I still think this linkage needs to exist here.

Reply:
This is an artifact of merging our two commits thoughtlessly. I think this is a fine idea.